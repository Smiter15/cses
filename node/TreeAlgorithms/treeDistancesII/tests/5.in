10
9 10
3 10
4 10
3 7
6 5
1 8
2 3
10 6
3 8
