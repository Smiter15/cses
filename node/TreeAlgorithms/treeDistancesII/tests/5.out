29 23 15 23 29 21 23 21 23 15 
